docs/sidebars.js

```diff
@@ -38,8 +38,8 @@ module.exports = {
         'sources/facebook',
         'sources/instagram',
         'sources/google',
-        'sources/sms-twilio',
         'sources/whatsapp-twilio',
+        'sources/sms-twilio',
         'sources/viber',
```

Comment:
change order of sms twilio in the sidebar so that it is exactly the same as the source list in the `sources/introduction`